<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComDialog</class>
 <widget class="QDialog" name="ComDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Rendering</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Avogadro::HistogramWidget" name="histogramWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cube</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cubesComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Enable volume rendering</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="enableVolumeRendering">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Enable surface rendering</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="enableIsosurface">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Isovalue</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="isoValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Opacity</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="opacity">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Avogadro::HistogramWidget</class>
   <extends>QWidget</extends>
   <header>histogramwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>enableIsosurface</sender>
   <signal>toggled(bool)</signal>
   <receiver>isoValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableIsosurface</sender>
   <signal>toggled(bool)</signal>
   <receiver>opacity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
